src/components/sidebar/common/SectionResult.js

```diff
@@ -22,10 +22,19 @@ export default function SectionResult({ addSection, section, disabled }) {
   const classes = useStyles();
 
   const message = 'Class successfully added';
+  const handleUndoClick = () => {
```

Comment:
I see why this could be difficult to test, since you want to grab the Button so that you can call its `onClick` but it's hard to grab the button because it's passed as a parameter in some other handler. One easy way to refactor this so it's easier to test is to break out the button into its own component. In this case I don't think we really need to do that - if it's complaining about coverage percentage, we can lower it. The numbers I picked when I lowered the coverage were a complete guess so perhaps it was a bit high. You can do that as part of this PR (look at jest.config.js)

Reply:
If we lower function coverage to below 75, it would work

Reply:
Got it. Thanks!